apache-ansible-demo/docs/prerequisites.md

```diff
@@ -0,0 +1,9 @@
+To run this resource package, you require the following prerequisites:
+
+1. An OpenStack environment configured with a user, an associated project, a deallocated floating IP, an Ubuntu 18.04 QCOW2 images and a *Key Pair* called *default* available on the target project.  
+2. For the Adopt lifecycle transition, an OpenStack stack in one of the following state: 'CREATE_COMPLETE','ADOPT_COMPLETE','RESUME_COMPLETE','CHECK_COMPLETE','UPDATE_COMPLETE'.  
+3. * A flavor of specification m1.small defined in the OpenStack project. The flavor defines the compute, memory, and storage capacity of a virtual server, also known as an instance.
```

Comment:
Quite trivial - but there is a bullet point after the point 3 which is not necessary.